### PR TITLE
 USWDS - Radio Button - radio shows unchecked when disabled even when checked

### PR DESCRIPTION
**Pull request by ghost**
Fix for https://github.com/uswds/uswds/issues/3979

## Description

When a radio input was disabled but also had the checked attribute there was no visual indicator of this combination of states. This PR adds styling to "fill-in" the radio input as if it were checked while also showing other styling indicating it's disabled state. Also adds examples on the radio input component of this :disabled:checked state, including an `.usa-radio__input--tile` variant example.

### CODE REVIEW

File: src/components/radio-buttons/radio-buttons--tile.njk [hunk 1]
```diff
@@ -17,5 +17,10 @@
       <input class="usa-radio__input usa-radio__input--tile" id="historical-carver-2" type="radio" name="historical-figures-2" value="george-washington-carver" disabled>
       <label class="usa-radio__label" for="historical-carver-2">George Washington Carver</label>
     </div>
+      <div class="usa-radio">
+      <input class="usa-radio__input usa-radio__input--tile" id="historical-marshall-2" type="radio" name="historical-figures" value="thurgood-marshall" disabled checked>
+      <label class="usa-radio__label" for="historical-marshall-2">Thurgood Marshall</label>
+    </div>
+
```
Comment (mejiaj) [inline src/components/radio-buttons/radio-buttons--tile.njk:24]: Fix indent and remove extra line on 24.

File: src/components/radio-buttons/radio-buttons--tile.njk [hunk 2]
```diff
@@ -17,5 +17,10 @@
       <input class="usa-radio__input usa-radio__input--tile" id="historical-carver-2" type="radio" name="historical-figures-2" value="george-washington-carver" disabled>
       <label class="usa-radio__label" for="historical-carver-2">George Washington Carver</label>
     </div>
+      <div class="usa-radio">
+      <input class="usa-radio__input usa-radio__input--tile" id="historical-marshall-2" type="radio" name="historical-figures" value="thurgood-marshall" disabled checked>
```
Comment (mejiaj) [inline src/components/radio-buttons/radio-buttons--tile.njk:21]:
```suggestion
        <input class="usa-radio__input usa-radio__input--tile" id="historical-marshall-2" type="radio" name="historical-figures-2" value="thurgood-marshall" disabled checked>
```

**Review by mejiaj — CHANGES_REQUESTED** [review]
Nice job! Just some minor comments

File: src/components/radio-buttons/radio-buttons--tile.njk (hunk 1 above)
Comment (thisisdano) [reply src/components/radio-buttons/radio-buttons--tile.njk:24]: @jeffffeffff there is still an extra line on line 24

File: src/components/radio-buttons/radio-buttons--tile.njk (hunk 2 above)
Comment (thisisdano) [reply src/components/radio-buttons/radio-buttons--tile.njk]: @jeffffeffff The `name="historical-figures-2"` change still isn't done

**Comment by thisisdano** [comment]
@mejiaj Sorry, I dismissed your review, prematurely determining it was stale. Yet it was not!

**Comment by mejiaj** [comment]
Closing in favor or more up-to-date PR.
https://github.com/uswds/uswds/pull/4160

**Pull request closed without merging**